File: Latihan 3/PHP/Ram.php
<?php
/* Saya Silmi Aulia Rohmah mengerjakan Latihan 3 DPBO 2022 C1 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */


class Ram{

    //atribut private 
    private $capacity;
    private $price;

    public function __construct(){ //konstruktor kosong

    }

    //metode getter dan setter

    public function getCapacity() {
		return $this->capacity;
	}

	public function setCapacity($capacity) {
		$this->capacity = $capacity;
	}

    public function getPrice() {
		return $this->price;
	}

	public function setPrice($price) {
		$this->price = $price;
	}

    //untuk print atribut kelas 
    public function PrintRam(){
        echo "======= RAM ========="."<br/>";
        echo " Capacity     : ". $this->getCapacity(). " GB"."<br/>";
        echo " Price : ". $this->getPrice()."<br/>";
        
    } 
    
	public function __destruct(){ //destruktor
		
	}
}

?>

File: Latihan 3/JAVA/Ram.java
/* Saya Silmi Aulia Rohmah mengerjakan Latihan 3 DPBO 2022 C1 
dalam mata kuliah DPBO untuk keberkahanNya maka saya tidak melakukan 
kecurangan seperti yang telah dispesifikasikan. Aamiin */

public class Ram{

    //atribut private 
    private int price = 0;
    private int capacity = 0;


    public Ram(){ //konstruktor kosong

    }

    //metode getter dan setter
	public int getPrice() {
		return this.price;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getCapacity() {
		return this.capacity;
	}

	public void setCapacity(int capacity) {
		this.capacity = capacity;
	}


    //untuk print atribut kelas 
    public void PrintRam(){

        System.out.println("======= RAM =========");
        System.out.println(" Capacity : " +  getCapacity() + " GB");
        System.out.println(" Price    : " + getPrice());
        
    } 

}